### PR TITLE
+doc #19796 explain how to obtain a JsValue using entity(as)

### LINKED ISSUE #19796: Document that entity(as[JsValue]) works
Since people were doing some weird things to get "raw json".

### PR DESCRIPTION
**Pull request by ktoso**
Follows up a twitter discussion and resolves https://github.com/akka/akka/issues/19796

### CODE REVIEW

**Comment by ktoso** [comment]
For reference https://twitter.com/jaceklaskowski/status/698521650745241600

**Comment by drewhk** [comment]
LGTM

File: akka-docs/rst/scala/http/routing-dsl/directives/marshalling-directives/entity.rst [hunk 1]
```diff
@@ -49,3 +49,9 @@ class.  It utilizes ``SprayJsonSupport`` via the ``PersonJsonSupport`` object as
 .. includecode2:: ../../../../code/docs/http/scaladsl/server/directives/MarshallingDirectivesExamplesSpec.scala
    :snippet: example-entity-with-json
 
+It is also possible to use the ``entity`` directive to obtain raw ``JsValue`` (spray-json) objects, by simply using
```
Comment (johanandren) [inline akka-docs/rst/scala/http/routing-dsl/directives/marshalling-directives/entity.rst]: Include a link to spray json perhaps?

**Comment by johanandren** [comment]
LGTM

**Pull request merged**